libs/core/mappers/aired.mapper.ts

```diff
@@ -0,0 +1,17 @@
+import { AiredDto } from '../dtos/aired.dto';
+import { Aired } from '../models/aired';
+
+export namespace AiredMapper {
+
+	/**
+	 * Map aired dto to model.
+	 * @param aired - Aired dto.
+	 * @returns Aired model.
+	 */
+	export function fromAiredDto(aired: AiredDto): Aired {
```

Comment:
Since we're using the mapper through the namespace there is no need to be that explicit with the function name.
```suggestion
	export function fromDto(aired: AiredDto): Aired {
```

Usage

```typescript
AiredMapper.fromDto(dto);
```